src/Responses/CreateMeetingResponse.php

```diff
@@ -101,9 +101,9 @@ public function getCreationDate()
     }
 
     /**
-     * @return true
+     * @return bool
      */
-    public function hasUserJoined()
+    public function hasUserJoined(): bool
```

Comment:
Should we add type hints for params and return type for the whole API?

Reply:
If you bump the error level in psalm.xml, it tries to infer more types, but this usually results in a lot of additional work. If you have time, it would be great.